plugins/cohere/credentials.go

```diff
@@ -0,0 +1,111 @@
+package cohere
+
+import (
+	"context"
+
+	"encoding/json"
+	"fmt"
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func Credentials() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.Credentials,
+		DocsURL:       sdk.URL("https://docs.cohere.com/reference/config"),
+		ManagementURL: sdk.URL("https://dashboard.cohere.ai/"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Authtoken,
+				MarkdownDescription: "JWT used to authenticate to Cohere stored in disk.",
+				Secret:              true,
+			},
+			{
+				Name:                fieldname.Email,
+				MarkdownDescription: "Email used to authenticate to Cohere.",
+				Optional:            true,
+			},
+			{
+				Name:                fieldname.URL,
+				MarkdownDescription: "URL of the operator server",
+				Optional:            false,
+			},
+		},
+		DefaultProvisioner: provision.TempFile(
+			cohereJSON,
+			provision.AtFixedPath("~/.command/config")),
+		Importer: importer.TryAll(
+			TryCohereConfigFile(),
+		)}
+}
+
+func TryCohereConfigFile() sdk.Importer {
+	return importer.TryFile("~/.command/config", func(ctx context.Context, contents importer.FileContents, in sdk.ImportInput, out *sdk.ImportAttempt) {
+		var config Config
+		if err := contents.ToJSON(&config); err != nil {
+			out.AddError(err)
+			return
+		}
+
+		if config.Contexts[config.CurrentURL].JWT == "" {
+			return
+		}
+
+		out.AddCandidate(sdk.ImportCandidate{
+			Fields: map[sdk.FieldName]string{
+				fieldname.URL:       config.CurrentURL,
+				fieldname.Authtoken: config.Contexts[config.CurrentURL].JWT,
```

Comment:
Any reason to save this under `authtoken`, instead of `jwt`?

If JWT is not already in `fieldname`s, feel free to add it!